ppr-ui/src/App.vue

```diff
@@ -210,6 +195,13 @@ export default class App extends Mixins(AuthMixin) {
     return process.env.ABOUT_TEXT
   }
 
+  /** Get banner text. */
+  private get bannerText (): string {
+    const bannerText: string = getFeatureFlag('banner-text')
+    // remove spaces so that " " becomes falsy
+    return bannerText?.trim() || null
+  }
+
```

Comment:
Small heads up here, i've a PR open which reformats the whole `App.vue` script to composition api script setup (from a class component) as part of our prep for the migration to V3. 
There will certainly be some conflicts.

Reply:
When do you plan to merge your PR? I can wait for it and solve all conflicts.

Reply:
If this PR needs to get in asap, i could also hold on my PR and resolve the conflicts. 
Whatever you prefer!

Reply:
PR is merged, will be some minor conflicts with the version and App.vue. 

The code in the template should be a-ok, just the computed values are structured a bit different in the script setup for composition api. Lots of examples in App.vue though :) 
Lmk if there is any issues!